### DISCUSSION TITLE
Displaying Custom Persian Font in GitHub README: Is It Possible?

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#65046] [Other Feature Feedback, Questions, & Ideas] [answered]
How can I set up a font on GitHub to be displayed, especially in the README? Is it possible to do this? I want to use a custom Persian font.

**Comment by arafajs** [comment, marked as answer]
Unfortunately, it is not possible to set up a custom font on GitHub to be displayed in the README. GitHub uses a fixed-width font for all READMEs, regardless of the user's preferences. This is done to ensure that the READMEs are consistent and readable across all platforms and devices.

If you want to use a custom Persian font in your README, **you can do so by embedding the font as an image.** This is not ideal, as it will increase the file size of your README and may not render properly on all devices.

**Reply by alidehkhodaei** [reply]
Can I change the font for my entire account page?

**Reply by arafajs** [reply]
Unfortunately, you cannot change the font

**Reply by alidehkhodaei** [reply]
Thank you

**Reply by truongtran456** [reply]
ok
